### ISSUE TITLE
Themable?

### ISSUE THREAD

**Issue by Eonasdan** [lukejacksonn/blog#2]
What's the themable-ness of this project? Do you have any guidelines one what would or wouldn't work?

Also you might want to consider creating a template repo without your blog posts in it.

**Comment by lukejacksonn** [comment]
Both of these are great suggestions! I didn't know anyone was actually using it tbh.

I hadn't really planned for theming but it would be possible just by tweaking some CSS values. If you do this and manage to nicely abstract what people might want to configure then perhaps we could define some CSS variables to make it easier for others to do the same in the future.

**Comment by Eonasdan** [comment]
I did manage to extra the css from the index and I'd be happy to provide that to you either as a c&p or a pr.

Thanks for the nice little project, it is very fast and works well for what it is. Unfortunately it's not flexible enough for me. There's no page meta for instance and that's not very good for SEO and what not.

I'd wish for three things:

* It'd be nice if the html part was not part of index.js and instead picked up a partial html file. I'm not familiar enough with your dispatching etc to make that work. I think this would allow further customization but the difficulty of doing some of the injection might be a barrier. Adding something like disqus or utterance is a consideration here. Would something like that even work if we're just injecting html.
* Post meta data: Setting the page title seems trivial enough, just find the first h1 at the top of the post. I started down this path but  was realizing I'd like `<meta property="og:title" content="meta title"/>` type of tags which would be very difficult without some other json file or magic in the markdown translation.
* I like that it attempts to load the github avatar however if you use custom domains those checks for the domain no longer work. This sorta ties into the previous point about a meta json file. Perhaps the username could be defined there along with post meta.

I totally understand not wanting to take on these ideas so no worries if this is just mostly a project for you. If you think that you'd like these features and we could work on it together I'd be down for that. My alternative is to use mkdocs to generate a static site but that's very bulkly for a simple blog.
